src/main/java/com/passpnu/passwordmanager/entity/ServiceEntity.java

```diff
@@ -18,16 +17,18 @@
 @NoArgsConstructor
 @Getter
 @Setter
-@Table(name="service")
+@Table
```

Comment:
Better revert to having `(name="service")`. Without Service CRUD it is quite difficult to assess but it may be useful. Also it is more readable

Reply:
Fixed